README.md

```diff
@@ -336,8 +336,6 @@ then update all package versions with:
 yarn run updatePackageVersion
 ```
 
-After that update the versions of all packages in the root `package.json`
```

Comment:
We don't need to do that anymore, instead we are adding version back to the root `package.json` and let `lerna version` handle dependencies automatically like it does all other versioned packages.